org.ghidra_sre.Desktop.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "org.ghidra_sre.Desktop",
+    "branch": "stable",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "19.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "command" : "ghidra",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+         "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin",
+         "--env=JAVA_HOME=/app/jdk"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+              "/usr/lib/sdk/openjdk11/installjdk.sh"
+            ]
+        },
+        {
+            "name": "ghidra",
+            "buildsystem": "simple",
+            "build-commands": [
+                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
+                "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
+                "install -Dm755 ghidra.sh /app/bin/ghidra",
+                "install -Dm644 org.ghidra_sre.Desktop.png /app/share/icons/org.ghidra_sre.Desktop.png",
+                "install -Dm644 org.ghidra_sre.Desktop.desktop /app/share/applications/org.ghidra_sre.Desktop.desktop",
+                "install -Dm644 org.ghidra_sre.Desktop.appdata.xml /app/share/appdata/org.ghidra_sre.Desktop.appdata.xml"
+            ],
+            "sources" : [
+                {
+                    "type": "archive",
+                    "sha256": "3d61de711b7ea18bdee3ed94c31429e4946603b3e7d082cca5e949bbd651f051",
+                    "url": "https://ghidra-sre.org/ghidra_9.1-BETA_DEV_20190923.zip"
+                },
+                {
+                    "type": "file",
+                    "path": "org.ghidra_sre.Desktop.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "org.ghidra_sre.Desktop.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html